### ISSUE TITLE
Daily scrums

### ISSUE THREAD

**Issue by lcabrini** [borderitsolutions/amadaa-server#17]
@BorderITS, @lexenington we have discussed about daily scrums and how to track them. I think we should submit our daily scrums each morning. But how and where?

Since we don't meet up each day, the stand-up wouldn't work. Another idea is via email. But to where, and how to we use this to track if we are meeting our targets?

**Comment by BorderITS** [comment]
I think when we do it in Github it will help us to keep track of each daily srcum 

1 proposal

**Comment by lcabrini** [comment]
Yes, I'm also in favor of keeping it here on github. But how do we best do it?

**Comment by BorderITS** [comment]
Each morning we post our daily srcum least by 10am to 11am

**Comment by lcabrini** [comment]
Yes, it's a good way to start the day. But where? As an issue, raised each day? On the wiki? And how do we keep a track of this?

**Comment by BorderITS** [comment]
Hmmm... We Will have to figure it out

**Comment by BorderITS** [comment]
Nelson Ocloo  my daily scrum  
 
Yesterday I was able to create more user stories and also made some some contribution on how the sales field will look like 

Today I will I try to create some user stories and also will make some opinions on other issues
